docusaurus.config.js

```diff
@@ -137,9 +137,6 @@ const config = {
                             {
                                 label: 'Github',
                                 to: 'https://github.com/WasmEdge/WasmEdge',
-                            }, {
-                                label: 'Dev docs',
-                                to: '/docs',
```

Comment:
Everything works like charm now, but could you tell why the section 140-142 was deleted.
I have test it locally and it's works fine, but I don't know the reason to why?

Could you try explaining?
was `to:/docs` breaking everything?

Reply:
> Everything works like charm now, but could you tell why the section 140-142 was deleted.
> I have test it locally and it's works fine, but I don't know the reason to why?
>
>Could you try explaining?
>was `to:/docs` breaking everything?

As mentioned earlier in the issue it was acting as baseUrl which results in broken link for other locales.

Like the `/docs` is already a baseUrl so at the time of building i18n it was being replace with the same thing like `/docs/zh/docs` which didn't existed in docs.

>Also pardon me for the unnecassary notifications which have been cause by my comments, they have been solved so I deleted them.

No worries.

> Also, could you try linking this PR in the issue, would be helpful and prevent anyone from accidentally addressing the issue and causing conflicts.

This can be easily prevented if there was checks on PR. I will create a workflow to address this so that in future thinks like this won't slip away in main branch.

Reply:
Thanks for the explanation, it has cleared my doubts by much now.

But what if a person wants to redirect user to /docs-->baseUrl by placing a link in footer, how he/she can do so?

Reply:
> But what if a person wants to redirect user to /docs-->baseUrl by placing a link in footer, how he/she can do so?

By using `/`.

Reply:
It now ends up on this error along with using `.` or `./`

```bash
[ERROR] Error: Cannot find module '@docusaurus/useBaseUrl'
Require stack:
- /home/bhardwaj/Code/Github/Wasmedge_docs/docusaurus.config.js
- /home/bhardwaj/Code/Github/Wasmedge_docs/node_modules/@docusaurus/core/lib/server/config.js
- /home/bhardwaj/Code/Github/Wasmedge_docs/node_modules/@docusaurus/core/lib/server/index.js
- /home/bhardwaj/Code/Github/Wasmedge_docs/node_modules/@docusaurus/core/lib/commands/build.js
- /home/bhardwaj/Code/Github/Wasmedge_docs/node_modules/@docusaurus/core/lib/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1026:15)
    at Function.Module._load (node:internal/modules/cjs/loader:871:27)
    at Module.require (node:internal/modules/cjs/loader:1098:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/home/bhardwaj/Code/Github/Wasmedge_docs/docusaurus.config.js:6:33)
    at Module._compile (node:internal/modules/cjs/loader:1196:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1250:10)
    at Module.load (node:internal/modules/cjs/loader:1074:32)
    at Function.Module._load (node:internal/modules/cjs/loader:909:12)
    at Module.require (node:internal/modules/cjs/loader:1098:19)
[INFO] Docusaurus version: 2.4.0
Node version: v16.20.0
~/Code/Github/Wasmedge_docs main !1 ?1 ❯ npm install                                           6s

up to date, audited 1104 packages in 13s

200 packages are looking for funding
  run `npm fund` for details

25 vulnerabilities (9 moderate, 16 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
~/Code/Github/Wasmedge_docs main !1 ?1 ❯ npm run build                                        13s

> book@0.0.0 build
> docusaurus build

[ERROR] Error: Cannot find module '@docusaurus/useBaseUrl'
Require stack:
- /home/bhardwaj/Code/Github/Wasmedge_docs/docusaurus.config.js
- /home/bhardwaj/Code/Github/Wasmedge_docs/node_modules/@docusaurus/core/lib/server/config.js
- /home/bhardwaj/Code/Github/Wasmedge_docs/node_modules/@docusaurus/core/lib/server/index.js
- /home/bhardwaj/Code/Github/Wasmedge_docs/node_modules/@docusaurus/core/lib/commands/build.js
- /home/bhardwaj/Code/Github/Wasmedge_docs/node_modules/@docusaurus/core/lib/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1026:15)
    at Function.Module._load (node:internal/modules/cjs/loader:871:27)
    at Module.require (node:internal/modules/cjs/loader:1098:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/home/bhardwaj/Code/Github/Wasmedge_docs/docusaurus.config.js:6:33)
    at Module._compile (node:internal/modules/cjs/loader:1196:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1250:10)
    at Module.load (node:internal/modules/cjs/loader:1074:32)
    at Function.Module._load (node:internal/modules/cjs/loader:909:12)
    at Module.require (node:internal/modules/cjs/loader:1098:19)
```

Reply:
> [ERROR] Error: Cannot find module '@docusaurus/useBaseUrl'

You haven't install your node modules correctly try installing them again.

Reply:
How to do that?

`npm run install`? or any other command?

Reply:
> Unable to fix these when I run `npm install`-
> 
> ```shell
> got  <11.8.5
> Severity: moderate
> Got allows a redirect to a UNIX socket - https://github.com/advisories/GHSA-pfrx-2q88-qq97
> No fix available
> ```
> 
> and this
> 
> ```shell
> trim  <0.0.3
> Severity: high
> Regular Expression Denial of Service in trim - https://github.com/advisories/GHSA-w5p7-h5w8-2hfq
> fix available via `npm audit fix`
> ```

Why are running `npm audit report`  and saying it's from `npm install`?

Reply:
When I run `npm install` it shows me list of dependencies which are outdated or deprecated, when trying to fix them using `npm audit fix` it does nothing.

Reply:
> When I run npm install it shows me list of dependencies which are outdated or deprecated, when trying to fix them using npm audit fix it does nothing.

This a not a place to ask question about npm. If you had problem running wasm-edge docs do ask about it. Just delete your existing repo and try fresh installing all dependenices.

Thank you!

Reply:
Ah okay, thanks!